tests/autest.sh

```diff
@@ -29,7 +29,7 @@ if [ ! -f ./env-test/bin/autest ]; then\
 # this is for rhel or centos systems
 test -r /opt/rh/rh-python35/enable && . /opt/rh/rh-python35/enable
 . env-test/bin/activate
-./env-test/bin/autest -D gold_tests $*
+./env-test/bin/autest -D gold_tests "$@"
```

Comment:
How is this going to work if the `env-test` directory isn't checked in? Is that pulled from somewhere else?